src/Components/LifecycleChart/LifecycleChart.tsx

```diff
@@ -304,20 +331,26 @@ const LifecycleChart: React.FC<LifecycleChartProps> = ({
         ariaTitle="Lifecycle bar chart"
         containerComponent={
           <ChartVoronoiContainer
-            labels={({ datum }: { datum: ChartDataObject }) => {
-              if (datum.name && datum.packageType && datum.y0) {
-                return `Name: ${datum.name}\nRelease: ${
-                  datum.version
-                }\nSupport Type: ${datum.packageType}\nSystems: ${
-                  datum.numSystems
-                }\nStart: ${formatDate(new Date(datum.y0))}\nEnd: ${formatDate(
-                  new Date(datum.y)
-                )}`;
-              }
-              return formatDate(new Date());
-            }}
-            labelComponent={<ChartTooltip constrainToVisibleArea />}
-            voronoiPadding={50}
+            labels={getTooltipLabel}
+            labelComponent={
+              <ChartTooltip 
+                constrainToVisibleArea 
+                flyoutWidth={240}
+                flyoutStyle={{ 
+                  fill: 'black',
+                  stroke: '#888',
+                  strokeWidth: 1,
+                  opacity: 0.95
+                }}
+              />
+            }
+            // Increase padding to make tooltip detection more generous on all sides
+            voronoiPadding={10}
```

Comment:
I think this was the main problem of https://issues.redhat.com/browse/RSPEED-810 Good catch!

Reply:
Yes exactly it took forever to figure out